### PR TITLE
added swu updates and multiplexing signal

### PR DESCRIPTION
**Pull request by ajb413**

### CODE REVIEW

File: signals/day3.js [hunk 1]
```diff
@@ -1,67 +1,39 @@
 opengrowth.signals.day3 = ( request, customer ) => {
```
Comment (raimondi1337) [inline signals/day3.js:1]: `customer` is no longer used

File: signals/day7.js [hunk 2]
```diff
@@ -1,47 +1,39 @@
 opengrowth.signals.day7 = ( request, customer ) => {
```
Comment (raimondi1337) [inline signals/day7.js:1]: `customer` is no longer used

File: signals/enable_multiplexing.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+opengrowth.signals.multiplexing = ( request, customer ) => {
```
Comment (raimondi1337) [inline signals/enable_multiplexing.js:1]: `customer` is no longer used

File: signals/signup.js [hunk 4]
```diff
@@ -1,18 +1,15 @@
 opengrowth.signals.signup = ( request, customer ) => {
```
Comment (raimondi1337) [inline signals/signup.js:1]: `customer` is no longer used

File: signals/uuid.js [hunk 5]
```diff
@@ -1,35 +1,40 @@
 opengrowth.signals.uuid = ( request, customer ) => {
```
Comment (raimondi1337) [inline signals/uuid.js:1]: `customer` is no longer used

File: handlers/signals.js [hunk 6]
```diff
@@ -40,7 +40,7 @@ export default request => {
     if (!email) return request.ok(); 
 
     // Process Customer Delight
-    return opengrowth.customer( email, signal ).then( customer => {
+    return opengrowth.customer.getCustomer( email, signal ).then( customer => {
```
Comment (raimondi1337) [inline handlers/signals.js:43]: `signal` isn't used

File: helpers/customer.js [hunk 7]
```diff
@@ -1,7 +1,9 @@
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 // Customer Fetch
 // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-opengrowth.customer = ( email, signal ) => {
+opengrowth.customer = {};
+// TODO: change .customer to .customer.getCustomer in open-growth/handlers/signals.js
```
Comment (raimondi1337) [inline helpers/customer.js:5]: This is done

File: helpers/customer.js [hunk 8]
```diff
@@ -1,7 +1,9 @@
...
+opengrowth.customer.getCustomer = ( email, signal ) => {
```
Comment (raimondi1337) [inline helpers/customer.js:6]: `signal` isn't used

File: signals/day3.js (hunk 1 above)
Comment (ajb413) [reply signals/day3.js:1]: l2r

**Pull request merged**